logcat/gradle.properties

```diff
@@ -1,3 +1,25 @@
-POM_ARTIFACT_ID=logcat
-POM_NAME=Logcat
-POM_PACKAGING=aar
+kotlin.code.style=official
+#kotlin.mpp.enableGranularSourceSetsMetadata=true
+#kotlin.native.enableDependencyPropagation=false
+kotlin.js.generate.executable.default=false
+android.useAndroidX=true
+
+# Required to publish to Nexus (see https://github.com/gradle/gradle/issues/11308)
+systemProp.org.gradle.internal.publish.checksums.insecure=true
+
+GROUP=com.squareup.logcat
```

Comment:
FWIW we do this configuration in build.gradle.kts in Okio and I prefer it there because it'll warn if I typo a field name.